src/main/java/aquarium/shop/order/PurchasedProduct.java

```diff
@@ -0,0 +1,58 @@
+package aquarium.shop.order;
+
+import aquarium.shop.product.Product;
+
+import javax.persistence.*;
+
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Table(name = "purchased_product")
+public class PurchasedProduct {
+
+
+    @Id
```

Comment:
Minor: maybe use sequence generator also here to make it common with other entities.